### DISCUSSION TITLE
Different undergowth with Rusty-pullauta versus Official pullauta

### DISCUSSION THREAD

**Discussion by agustincaballero** [karttapullautin/karttapullautin#26] [General]
Hi, I am part of the team that has created Mapant Spain (https://mapant.es). First of all thank you for Rusty-pullauta, it is great!!

In order to generate the map, we are currently using the official pullauta release (the one available at https://www.routegadget.net/karttapullautin/); we are now evaluating to use Rusty-pullauta.

Everything is working great except for one detail: the undergrowth generated by Rusy-pullauta is quite different to the other one (generally less undergrowth generated).

Is this an observed and expected behaviour? Or are we missing something?

I attach two images with the undergrowth generated by both in a sample zone, and also links to the pullauta.ini used and the .LAZ file used. 
https://mapant.es/ficheros/pullauta.ini
https://mapant.es/ficheros/in.laz
![undergrowth_official](https://github.com/rphlo/rusty-pullauta/assets/18677366/4cdd2975-99ef-4d02-9ef6-64a197956fe3)
![undergrowth_rusty](https://github.com/rphlo/rusty-pullauta/assets/18677366/ed2b4818-1f29-4944-a997-9efdcebef40d)

**Comment by stefankinell** [comment]
I can only say that when I tested this during the spring, I did not look so much at the undervegetation output. In the areas I use it, there is not a lot of undergrowth and therefore I have not paid attention to the difference. So if you found something that differs, it is most likely a bug in the new code.

**Comment by stefankinell** [comment]
BTW @agustincaballero , are you doing a rerun with vegetation only for Spain? Have you seen my branch where I have looked at how a vegetation rerun in batch mode can be executed faster without doing the contours and cliffs? 

I am planning a rerun of all vegetation of https://kartor.gokartor.se over Sweden, and the first alterations I did to the script saved me about half the time. Which is nice when you are planning a rerun of the whole country :-D

**Comment by agustincaballero** [comment]
Hi, Stefan. We are processing all Spain again because we have a completely new set of LAZ data; also, the vector data (roads, buildings, ...) is also updated regularly, so that is the reason why we process everything again.
But thanks for that option of rerunning only for vegetation, it might be very handy some time!!

**Reply by stefankinell** [reply]
I see,
At some point it would be interesting to see if we can learn some tricks from each other, all the projects that are doing nation wide maps. But perhaps we should leave that out of this discussion so I do not drift away to much...

**Comment by rphlo** [comment]
This is fixed in release 20231118a, thanks for the feedback.
